lib/src/hydrated_bloc_storage.dart

```diff
@@ -36,8 +37,12 @@ class HydratedBlocStorage implements HydratedStorage {
 
     if (await file.exists()) {
       try {
-        storage =
-            json.decode(await file.readAsString()) as Map<String, dynamic>;
+        String source = await file.readAsString();
+        if (source.isEmpty) {
+          await file.delete();
```

Comment:
why do we need to delete the file when source is empty?

Reply:
I think it was there as cleanup.